modules/monitoring/provider.tf

```diff
@@ -0,0 +1,4 @@
+provider "aws" {
+  alias  = "us-east-1"
+  region = "us-east-1"
```

Comment:
hm, is this supposed to be here?

Reply:
It should be here, since we use it as a parameter to set the route53 health check region and the alarm related to that healthcheck. Otherwise will create the alarm in the default region for example, eu-central-1 and the check by default is deployed in us-east-1 and they cannot be connected inter-region.